redi-connect-front/src/components/atoms/Button.tsx

```diff
@@ -40,6 +47,23 @@ const Button = ({
   )
 }
 
+Button.Nav = ({
+  children,
+  to,
+  size = 'small'
+}: NavProps) => {
+  return (
+    <NavLink
+      to={to !== undefined ? to : '/'}
```

Comment:
As far as I see this will never be the case. So for the sake of simplicity remove the condition.

Reply:
`Button.Nav` integrated into `Button`